freedomofpress/securedrop-workstation
# Rename VMs for clarity

Renames the following:
- `sd-svs` -> `sd-app`
- `sd-export-usb` -> `sd-devices`
- `sd-svs-disp` -> `sd-viewer`

See #285 for background and discussion.

Resolves #285. (Not changing `buster` naming convention for cost/benefit reasons.)

Dependent PRs:
- https://github.com/freedomofpress/securedrop-export/pull/49
- https://github.com/freedomofpress/securedrop-proxy/pull/59
- https://github.com/freedomofpress/securedrop-client/pull/701
- https://github.com/freedomofpress/securedrop-debian-packaging/pull/128

# Status

Ready for review

# Deployment

Once all PRs are merged, `make clean && make all` should do the trick, but the client will only work if you are using a dev build or a nightly that incorporates the changes.

# Testing

1. `make clean && make all` on this branch (make sure you have no conflicting `sd-app` VM in place)
2. Verify that all VMs are named as described.
3. Build a new `securedrop-client` package on the `vm-rename` branch of `securedrop-client`, and install it in the `sd-app-buster-template`. Restart `sd-app` to pick up the change.
4. Run the client. Verify that you can successfully log in, download, open, and export files.
